### PR TITLE
feat: add recipe for red velvet hot chocolate drink

### PR DESCRIPTION
**Pull request by savioxavier**
## Description

This PR adds the following:
- New chocolate based drink called "Red Velvet Hot Chocolate"
- Add recipe image for the above

### CODE REVIEW

File: src/recipes/red-velvet-hot-chocolate.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+            "measure": "tablespoon",
+            "ingredient": "unsweetened cocoa powder"
+        },
+        {
+            "quantity": "1",
+            "measure": "tablespoon",
+            "ingredient": "pure vanilla extract"
+        },
+        {
+          "quantity": 1,
```
Comment (alfg) [inline src/recipes/red-velvet-hot-chocolate.json:42]: Should be a string.

**Review by alfg — CHANGES_REQUESTED** [review]
Please fix the recipe json. Thanks.

File: src/recipes/red-velvet-hot-chocolate.json (hunk 1 above)
Comment (savioxavier) [reply src/recipes/red-velvet-hot-chocolate.json]: Should be fixed in https://github.com/alfg/opendrinks/pull/1309/commits/7a888eb66952dbd654343277742a9b9857335b36

**Comment by savioxavier** [comment]
@alfg Should be fixed now. Let me know if there's anything else to be done.

**Pull request merged**